subiquity/server/server.py

```diff
@@ -663,7 +663,38 @@ def validate_autoinstall(self):
                 context=ctx,
             )
 
-    def load_autoinstall_config(self, *, only_early):
+    @with_context(name="read_config")
+    def _read_config(self, *, cfg_path: str, context: Context) -> dict[str, Any]:
+        with open(cfg_path) as fp:
+            config: dict[str, Any] = yaml.safe_load(fp)
+
+        autoinstall_config: dict[str, Any] = dict()
+
+        # Support "Autoinstall" as a top-level key
+        if "autoinstall" in config:
+            autoinstall_config = config.pop("autoinstall")
+
+            # but the only top level key
+            if len(config) != 0:
+                self.interactive = bool(autoinstall_config.get("interactive-sections"))
+                msg: str = (
+                    "autoinstall is not a valid cloud config datasource.\n"
```

Comment:
```suggestion
                    "autoinstall.yaml on the ISO is not a valid cloud config datasource.\n"
```
or something like that

Reply:
I think just "autoinstall.yaml is not..." is sufficient. I don't think it's likely but I don't want users to think autoinstall elsewhere is a valid cloud config datasource. What do you think?